src/main/java/com/synopsys/integration/alert/workflow/scheduled/update/UpdateChecker.java

```diff
@@ -67,10 +72,28 @@ public UpdateModel getUpdateModel() {
         final IntHttpClient intHttpClient = createHttpClient();
         final DockerTagRetriever dockerTagRetriever = new DockerTagRetriever(gson, intHttpClient);
 
-        final String currentVersion = aboutReader.getProductVersion();
-        final Optional<String> optionalLatestAvailableVersion = getLatestAvailableVersion(dockerTagRetriever);
+        final AboutModel aboutModel = aboutReader.getAboutModel();
+        final String currentVersion = aboutModel.getVersion();
+        final String alertCreated = aboutModel.getCreated();
+
+        final Boolean isProduction = isProductionVersion(currentVersion);
+
+        final DockerTagModel latestAvailableVersion = getLatestAvailableTag(dockerTagRetriever, isProduction);
         final String repositoryUrl = dockerTagRetriever.getRepositoryUrl();
-        return new UpdateModel(currentVersion, optionalLatestAvailableVersion.orElse(null), repositoryUrl);
+
+        return getUpdateModel(currentVersion, alertCreated, latestAvailableVersion, repositoryUrl);
+    }
+
+    public UpdateModel getUpdateModel(final String currentVersion, final String alertCreated, final DockerTagModel latestAvailableVersion, final String repositoryUrl) {
+        int comparison = compareVersions(currentVersion, latestAvailableVersion.getName());
```

Comment:
why not use the comparator from getTagOrder here? appears to be the same comparison